### PR TITLE
Add updated TTK in-situ with Catalyst tutorial

### PR DESCRIPTION
**Pull request by pierre-guillou**
This is an updated version of the TTK in-situ with Catalyst tutorial, using the latest releases of the following sofware:

- TTK 0.9.7
- ParaView 5.6.0
- Mesa 18.3.3
- Code_Saturne 5.0.9

This new tutorial is a drop-in replacement of the old one.
The old tutorial page was renamed and kept as it was, and a link in the new page should direct to it.

I tried to improve the use of HTML tags with constructs like ol/li or pre/code. A new CSS rule has been added for the latter.

Updated helper images have been inserted: 

- one for the ttk_pipeline.py patch code
- one for the new required step about renaming variables in the ParaView form generating ttk_pipeline.py (with a slight change of graphical theme...)

The provided ParaView PVSM pipeline and the preconfigured tarball have also been updated.

### CODE REVIEW

**Comment by julien-tierny** [comment]
thanks, looks good!
-- 
Dr Julien Tierny
CNRS Researcher
Sorbonne Universite
http://lip6.fr/Julien.Tierny

File: catalyst.html [hunk 1]
```diff
@@ -142,369 +142,410 @@ <h1>Topology ToolKit</h1>
...
-The configuration interface opens. Press <code>[c]</code> to configure. At this stage, under Linux,
-TTK's build should be automatically configured correctly by default so you can press <code>[g]</code> to
-generate.<br>
-<br>
+<pre><code>
+$ cd ~/ttk/ttk-0.9.7/
+$ mkdir build
+$ cd build/
+$ cmake                                                     \
+      -DCMAKE_PREFIX_PATH=/usr/local/lib/cmake/paraview-5.6 \
```
Comment (julien-tierny) [inline catalyst.html:371]: strange. cmake's finding that path correctly on all installs I experimented (gentoo, ubuntu)

Comment (pierre-guillou) [reply catalyst.html:371]: I retried yesterday with a new Ubuntu Desktop VM, and TTK CMake configuration script would still fail at https://github.com/topology-tool-kit/ttk/blob/v0.9.7/core/vtk/CMakeLists.txt#L49.

I think CMake overlooks the /usr/local/lib/cmake/paraview-5.6 folder when searching for VTKConfig.cmake.

This bug seems to be resolved in master.

Comment (julien-tierny) [reply catalyst.html:371]: my bad, I forgot you were using paraview's internal vtk for that (and not a system wide install of vtk).
makes sense then.

**Pull request merged**